ui/src/components/Exchange/ManualFairValue.scss

```diff
@@ -0,0 +1,28 @@
+
+@mixin manual-fair-value {
+    .manual-fair-value {
+        display: flex;
+        flex-direction: column;
+        background-color: var(--white);
+        border: 1px solid var(--cool-grey-90);
+        padding: $spacing-m;
+        margin-left: $spacing-m;
+        border-radius: 2px;
+        height: fit-content;
+        animation: fadeIn .5s;
```

Comment:
animations 👌

Reply:
ha, I think that was css I borrowed from Sarah elsewhere...my CSS is getting better but not quite "add animations" level 😄